### PR TITLE
Adds super speed super powers

### PR DESCRIPTION
**Pull request by LanCartwright**
Human speedsters are able to create a vibrating hand, and throwable icicles made of water molecules.
http://puu.sh/o9Mm0/cab87d0854.mp4 Dodging
http://puu.sh/o9MuK/cbb826aad7.mp4 Beating Beepsky at his own game.
http://puu.sh/o9MZL/c3f681e6d3.mp4 Vibrating hand through Syndicate operative.

New update tomorrow.

### CODE REVIEW

File: code/datums/diseases/velocity.dm [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+	var/list/spelllist = affected_mob.mob_spell_list.Copy()
+	if(affected_mob.mind)
+		spelllist |= affected_mob.mind.spell_list
+	for(var/S in spelllist)
+		if(istype(S, /obj/effect/proc_holder/spell/targeted/touch/vibrate))
+			return
+	affected_mob.mind.AddSpell(new /obj/effect/proc_holder/spell/targeted/touch/vibrate)
+
+/datum/disease/velocity/proc/grant_iciclehand()
+	if(affected_mob.dna.species.id != "human")//can't vibrate calws fast enough
```
Comment (Niknakflak) [inline code/datums/diseases/velocity.dm]: calws

**Comment by Niknakflak** [comment]
Seems fun in theory.  Super overpowered and awful for regular game use.
Probably just add it to the list of things Admins can only spawn and rarely if ever use.

File: code/datums/diseases/velocity.dm (hunk 1 above)
Comment (LanCartwright) [inline code/datums/diseases/velocity.dm]: whoops.

**Comment by LanCartwright** [comment]
I might, just "might" make something less powerful, useable for players.

No promises; even I can't balance this.

**Comment by lzimann** [comment]
Just make it op and admin-only without destroying in the balance meme process

**Comment by KazeEspada** [comment]
Get your human agenda out of here you limpdicked fucknugget.

File: code/datums/diseases/velocity.dm [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+			if(ishuman(affected_mob))
+				var/mob/living/carbon/human/H = affected_mob
+				H.unEquip(H.wear_suit)
+			affected_mob.equip_to_slot_or_del(new /obj/item/clothing/suit/armor/reactive/speed(affected_mob), slot_wear_suit)
+			grant_vibratinghand()
+			grant_iciclehand()
+			affected_mob.next_move_adjust = -3
+
+/datum/disease/velocity/proc/grant_vibratinghand()
+	if(affected_mob.dna.species.id != "human")//can't vibrate scales fast enough
```
Comment (KazeEspada) [inline code/datums/diseases/velocity.dm]: Remove this fucker.

File: code/datums/diseases/velocity.dm (hunk 1 above)
Comment (KazeEspada) [inline code/datums/diseases/velocity.dm]: Just remove the line.

File: code/datums/diseases/velocity.dm (hunk 2 above)
Comment (PKPenguin321) [inline code/datums/diseases/velocity.dm]: lizards btfo :boom:

Comment (KazeEspada) [inline code/datums/diseases/velocity.dm]: Human authority is a config option that defaults to off.

File: code/modules/mob/living/carbon/human/species.dm [hunk 3]
```diff
@@ -774,6 +774,10 @@
 		. -= 1
 	if(H.status_flags & GOTTAGOREALLYFAST)
 		. -= 2
+	if(H.status_flags & GOTTAGOGONE)
+		. -= 3
```
Comment (octareenroon91) [inline code/modules/mob/living/carbon/human/species.dm]: You can get this effect by putting GOTTAGOFAST and GOTTAGOREALLYFAST:

```
.-=1
.-=2
```

Someone will probably appreciate the spare bitflag slot.

File: code/modules/clothing/suits/armor.dm [hunk 4]
```diff
@@ -246,6 +246,58 @@
 			M.adjustFireLoss(25)
 			playsound(M, 'sound/machines/defib_zap.ogg', 50, 1, -1)
 		return 1
+
+//velocity 9 speed armour
+/obj/item/clothing/suit/armor/reactive/speed
+	name = "speed force insticts"
```
Comment (KorPhaeron) [inline code/modules/clothing/suits/armor.dm]: Make up your own name

File: code/datums/diseases/velocity.dm [hunk 5]
```diff
@@ -0,0 +1,87 @@
+/datum/disease/velocity
+	name = "Velocity 9 poisoning"
```
Comment (KorPhaeron) [inline code/datums/diseases/velocity.dm]: Make up your own name

File: code/modules/clothing/suits/armor.dm [hunk 6]
```diff
@@ -246,6 +246,58 @@
...
+				continue
+			turfs += T
+		if(!turfs.len)
+			turfs += pick(/turf in orange(dodge_range, H))
+		var/turf/picked = pick(turfs)
+		if(!isturf(picked))
+			return
+		H.forceMove(picked)
+		reactivearmor_cooldown = world.time + 20
+		return 1
```
Comment (KorPhaeron) [inline code/modules/clothing/suits/armor.dm]: This all looks pretty copy paste

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 7]
```diff
@@ -328,3 +328,21 @@
 		M.adjustOxyLoss(1, 0)
 	..()
 	. = 1
+
+/datum/reagent/medicine/velocity9
+	name = "Velocity 9"
```
Comment (KorPhaeron) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm]: Same as above. Make it "original"

**Comment by KorPhaeron** [comment]
Those dmi better not be flash costumes

**Comment by LanCartwright** [comment]
They aren't

**Comment by drynwyn** [comment]
The hell is with the "only humans get vibrating hand?" How does that make any sense? "can't vibrate scales fast enuff" is not a reason, it's an excuse. Given that this is and clearly always will be admin-only, there is no good reason to restrict features to humans.

**Comment by PKPenguin321** [comment]
LanCartwright does a lot of things wrong, but he always gets making lizard players salty oh-so right

**Comment by lzimann** [comment]
\>implying admins will give speed force to filthy unhumans

**Comment by drynwyn** [comment]
Race restrictions just don't make sense for an item that's already admin-only- if an admin wanted to give a lizard SPEEDYHANDS they could just varedit around the restrictions. If he didn't want a lizard to have SPEEDYHANDS, there's no way for him to get it.

**Comment by PKPenguin321** [comment]
the thing is that afaik this is just a disease that gives you spells, so if the admins gave the crew the disease to a lizard i suppose it could spread to a human. sure admins could give a lizard the spells straight up but just by itself admins could slap the disease on a lizard and still cause chaos, with no added confusion.

File: code/datums/diseases/velocity.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/datums/diseases/velocity.dm]: you #triggered the furry so leave it in

File: code/datums/diseases/velocity.dm (hunk 2 above)
Comment (Iamgoofball) [inline code/datums/diseases/velocity.dm]: you #triggered the furry so leave it in

**Comment by optimumtact** [comment]
Please dont' add more stuns

**Comment by LanCartwright** [comment]
For fuck sake who merged a PR that created a conflict.

**Comment by KorPhaeron** [comment]
>inhands
>onmob suits

I wouldn't bother solving the conflicts until the code is ready/approved. It will keep conflicting as long as its up.

**Comment by Xhuis** [comment]
The grammar in this pull request triggers me on a personal level.

**Comment by Jazaen** [comment]
>Giving superpowers to virology (Well, kind of), and not genetics, the department that is literally only about cloning, superpowers and occasional identity theft

**Comment by LanCartwright** [comment]
gotta start somewhere. I specialise in virology.

**Comment by Xhuis** [comment]
And nuke op guns.

**Comment by Cheridan** [comment]
Sorry but this is in violation of our reference policy.

**Comment by Dorsisdwarf** [comment]
@LanCartwright You doin this any more? It's in danger of reaching the Superfluous Zone

**Comment by GunHog** [comment]
I did not even know this existed! These all seem like Genetics things to me. Needs to not be hard-locked to humans, though.

**Comment by LanCartwright** [comment]
I'll fix up the conflict, maybe do some more. It's going stale because it's pretty memey.

I might, just might give it a process to trigger it; X and Y in body, radiation above Z, hit by particle accelerator, then shocked by Telsa to get powers.

**Comment by KorPhaeron** [comment]
@LanCartwright 

Any chance of this getting finished?

**Comment by LanCartwright** [comment]
after dismemberment I'll reopen this with fixed conflicts

**Comment by KorPhaeron** [comment]
@LanCartwright 

Find someone willing to review it before you go through the trouble of reopening it would be my suggestion, nobody seems to want to touch this

**Pull request closed without merging**